pkg/cmd/pulumi/preview.go

```diff
@@ -539,7 +539,7 @@ func newPreviewCmd() *cobra.Command {
 	}
 	cmd.PersistentFlags().StringVar(
 		&importFilePath, "import-file", "",
-		"Save any creates seen during the preview into an import file to use with `pulumi import`")
+		"Save any creates seen during the preview into an import file to use with pulumi import")
```

Comment:
```suggestion
		"Save any creates seen during the preview into an import file to use with 'pulumi import'")
```

Maybe having this in single quotes would help distinguish it a little better as being a shell command?